### PR TITLE
add image references

### PR DESCRIPTION
**Pull request by juanvallejo**
cc @deads2k

### CODE REVIEW

File: manifests/image-references [hunk 1]
```diff
@@ -0,0 +1,12 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-openshift-controller-manager-operator
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-cluster-openshift-controller-manager-operator:latest
```
Comment (sttts) [inline manifests/image-references]: latest? Don't we version them?

Comment (sttts) [reply manifests/image-references]: Do we still push to docker.io?

Comment (juanvallejo) [reply manifests/image-references]: Thanks, done

Comment (juanvallejo) [reply manifests/image-references]: I will do this for the other three operators as well

**Comment by deads2k** [comment]
@tnozicka something looks stuck here with travis.  

manually merging

**Comment by deads2k** [comment]
interesting.  it self-unstuck itself after I mucked with settings.

**Comment by tnozicka** [comment]
@deads2k sorry I missed this, cleaning the inbox now.

Why do we still have Travis? I don't have perms but I though last time we agreed to disable it on all of our repos - that's why I haven't made it explicitly optional yet.

**Pull request merged**